symfony.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Symfony
+layout: post
+permalink: /symfony
+link: https://symfony.com/roadmap
+activeSupportColumn: true
+command: composer show symfony/symfony | grep versions
+releases:
+  '2.3':
+    release: 2013-05-01
+    support: 2016-05-01
+    eol: 2017-05-01
```

Comment:
The spacing here looks incorrect, but it seems to parse correctly, so I'm merging for now.